src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -51,9 +51,13 @@ def create(input_dataset, feature="drawing", target="label",
     start_time = _time.time()
 
     if max_iterations == 0:
-        max_iterations = num_epochs * len(input_dataset) / batch_size
+        # @TODO: Be able to automatically choose number of epochs based on 
+        # data size
+        num_epochs = 100
+        num_iterations = num_epochs * len(input_dataset) / batch_size
     else:
         num_epochs = max_iterations * batch_size / len(input_dataset)
+        num_iterations = max_iterations
```

Comment:
If I'm not mistaken, our other toolkits (besides OD) use "iteration" to refer to a pass through the training data. For example if I specify, `max_iterations=100`, should I expect 100 passes through the training data? Or 100 mini-batches?

Reply:
The intention here is 100 mini-batches, but if our other toolkits use it to refer to number of epochs, I'll change the code to account for it.

Reply:
@nickjong Looks like all toolkits except for the Object Detector and Style Transfer take `max_iterations` to mean "Maximum number of Epochs". However, OD and Style Transfer take it to mean "Maximum number of training iterations ~ number of mini-batches". While the latter is more correct from an academic perspective, I have modified the drawing classifier to take `max_iterations` to mean "Maximum number of Epochs" for two reasons, after discussing with @ylow and @TobyRoseman :
1. An epoch is a more intuitive parameter to pass in for our users.
2. A default value for number of epochs makes more sense for this toolkit, just like it does for the Image Classifier.
3. To align with other toolkits, especially the Image Classifier.

It is a unfortunate that the same parameter name means different things for different toolkits. There are three possible solutions, all API-breaking changes outside the scope of this PR that we should seriously think about before executing:
1. Rename the parameter from `max_iterations` to `max_epochs` for all toolkits except ST/OD. 
2. Maybe change OD and ST logic to also take in number of epochs but that may be suboptimal for the respective toolkits. 
3. For OD and ST, rename the parameter from `max_iterations` to `max_mini_batches` or something distinct from `max_iterations` to not cause confusion.

Note that for any parameter renames, there is a backwards-compatibility supporting path through `kwargs`, as pointed out by @ylow .